react/package.json

```diff
@@ -8,15 +8,15 @@
     "react": "^16.8.6",
     "react-apollo": "^2.5.8",
     "react-dom": "^16.8.6",
-    "react-intl": "^3.11.0"
+    "react-intl": "2.9.0"
```

Comment:
Why go back a major? 🤔 
Or did the React builder do this?

Reply:
I've reverted the react-intl major because it was the one being used by rich-text . I thought it was necessary for them all to be on the same version, but now I think render always overrides it, right?

Reply:
Yep! The one from render will always be the one every app is actually using